### PR TITLE
chore(signup): refactor input component props data

### PR DESCRIPTION
**Pull request by Danielshow**
What does this PR do?
This PR removes curly braces from Input component props of type string

Description of Task to be completed?
- remove curly braces from prop-types of string
- remove `!important` keyword from css stylesheet

How should this be manually tested?
N/A

Any background context you want to provide?
N/A

What are the relevant pivotal tracker stories?
N/A

Screenshots (if appropriate)
N/A

### CODE REVIEW

File: src/containers/Auth/SignUp/style/signup.scss [hunk 1]
```diff
@@ -63,8 +63,8 @@
   .center a {
     text-decoration: underline;
   }
-  .inValid, .inValid:focus {
-    border:1px solid red;
+  .inValid {
+    border:1px solid red !important;
```
Comment (efalayi) [inline src/containers/Auth/SignUp/style/signup.scss]: Try to do this without `important`

File: src/containers/Auth/Login/Login.js [hunk 2]
```diff
@@ -108,6 +108,7 @@ export class Login extends Component {
               header="Welcome back! 👋"
               subHeader='“Today a reader, tomorrow a leader”'
               onSubmit={this.handleSubmit}
+              container_id='login-container'
```
Comment (theghostyced) [inline src/containers/Auth/Login/Login.js]: @Danielshow. `CamelCase` would have been better here.

File: src/containers/Auth/SignUp/SignUp.js [hunk 3]
```diff
@@ -128,47 +128,48 @@ export class SignUp extends Component {
     const subHeader = '“Today a reader, tomorrow a leader”';
     const { state: { error }, props: { isLoading } } = this;
     return (
-      <FormWrapper imageUrl={image} displayImage={true}>
+      <FormWrapper imageUrl={image} displayImage={true} img_id='img-responsive'>
         <FormContainer header={header} subHeader={subHeader}
-          onSubmit={this.handleSubmit}>
-          <div className="signup-container">
+          login_id='login-wrapper' onSubmit={this.handleSubmit}
```
Comment (theghostyced) [inline src/containers/Auth/SignUp/SignUp.js]: Same here @Danielshow. `CamelCase` would be better here

File: src/containers/Auth/Login/Login.js (hunk 2 above)
Comment (Danielshow) [reply src/containers/Auth/Login/Login.js]: ok, thanks

File: src/containers/Auth/SignUp/SignUp.js (hunk 3 above)
Comment (Danielshow) [reply src/containers/Auth/SignUp/SignUp.js]: Ok

**Pull request merged**